### ISSUE TITLE
Weekday offsets in dates not supported

### ISSUE THREAD

**Issue by simonpoole** [opening-hours/opening_hours.js#242]
The evaluator digest neither

`Jan 1-Fr`

not

`easter-Fr`

Both should be supported according to the spec.

**Comment by ypid** [comment]
I am not quite following and I think I have not seen this syntax before. Can you enlighten me? I checked the spec ([current](https://wiki.openstreetmap.org/wiki/Key:opening_hours/specification) and [previous](https://openingh.ypid.de/netzwolf_mirror/time_domain/specification.html) one) and must by missing something.

There is some other crazy syntax element which I have not yet implemented: https://github.com/opening-hours/opening_hours.js/issues/25

**Comment by ypid** [comment]
I am not quite following and I think I have not seen this syntax before. Can you enlighten me? I checked the spec ([current](https://wiki.openstreetmap.org/wiki/Key:opening_hours/specification) and [previous](https://openingh.ypid.de/netzwolf_mirror/time_domain/specification.html) one) and must be missing something.

> easter-Fr
> Both should be supported according to the spec (the later should be Good Friday).

```
easter -2 days open "Good Friday"
```

works for this.

Note that there are some other crazy features I have not yet implemented yet: https://github.com/opening-hours/opening_hours.js/issues/25

**Comment by simonpoole** [comment]
In the spec: 

<date_offset>       [ <plus_or_minus> <wday> ] [ <day_offset> ]

**Comment by ypid** [comment]
I have seen this, the thing is that you can not put `<date_offset>` anywhere alone without `<date_to>` before.

**Comment by simonpoole** [comment]
The relevant definitions are

`<monthday_range>`
is
`<date_from>` 
or
`<date_from> [ <date_offset> ] +` 
or
`<date_from> [ <date_offset> ] - <date_to> [ <date_offset> ]`

I was a bit lazy when opening the issue, according to the spec above it should only work in a range, but the evaluator doesn't work with that either. Naturally there is no real reason why the first variant isn't 
`<date_from> [ <date_offset> ] ` but that is unlikely to turn up irl in any case.

**Comment by ypid** [comment]
So you agree that `Jan 1-Fr` and `easter-Fr` are currently not covered by the spec? I have not yet seen a need for this to be honest.

**Comment by ypid** [comment]
True. I must say `Jan 1-Fr-Sep 30` looks a bit odd and I am sure people will hunt us for supporting that :-)
